posts/using-streamlit-chatgpt-and-snowflake-to-easily-bring-insights-to-end-users.md

```diff
@@ -0,0 +1,660 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Data
+- Streamlit
+- Snowflake
+- ChatGPT
+date: 2023-07-13T13:33:00.000Z
+title: "Using Streamlit and ChatGPT with Snowflake to Easily bring Data Insights to End Users"
+image: 
+---
+
+# Using Streamlit and ChatGPT with Snowflake to Easily bring Data Insights to End Users
+
+We're all seeing the boom in Generative AI over the past few weeks and months as the entire industry re-evaluates what potential use-cases for Generative AI are. So if you're a data engineer (like me), you might be wondering ***how AI can change how stakeholders and business decision makers interact with the data you curate and provide for them***. In this blog, I'm going to layout one potential approach for integrating Generative AI with your dashboards and democratizing data for less technical stakeholders.
+
+The core of the application that we're building is on [Streamlit](https://docs.streamlit.io/). Streamlit is an open-source Python library which make it easy to share web applications in minutes.
+
+## Prerequisites
+I started by installing Streamlit locally. As per the [Installation instructions](https://docs.streamlit.io/library/get-started/installation#install-streamlit-on-macoslinux) for MacOS/Linux, I created a python virtual environment, pip installed streamlit and tested that the installation worked by running `streamlit hello`.
+
+![Streamlit Hello Application](https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2023/07/streamlit-gpt-blog--streamlit-hello-app.png)
+
+A few other things you'll need for this demo are a Snowflake environment and ChatGPT credentials.
```

Comment:
maybe add a link to snowflake trial, as well as a link to how to generate necessary credentials / token for the chat gpt api